CHANGELOG.md

```diff
@@ -1,3 +1,107 @@
+## Solidus v3.3.0 (2023-01-24)
+
+## Solidus Core
+- Add coverage report badge using Codecov #3136 (@rubenochiavone)
```

Comment:
Do we need the links here as well?

Reply:
Thanks for pointing that out. For context, I thought Github automatically handled that on render time, but that's not the case, as it's clear from [their docs](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls#issues-and-pull-requests):

> Note: Autolinked references are not created in wikis or files in a repository.

I fixed it manually and created #4872 to fix it for the next time.